### ISSUE TITLE
Rename the master branch to main?

### ISSUE THREAD

**Issue by seisman** [gmt-china/GMT_docs#1094]
Maybe we should rename the master branch to main following https://github.com/github/renaming?

**Comment by seisman** [comment]
FYI, I've renamed the master branch to main. You may need to update your local repository using the following commands:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

**Comment by seisman** [comment]
Closed by 19026456d562ae8c6444936155cdb20f1e9359a5.
